sdk/features/arm-features/tsconfig.json

```diff
@@ -9,7 +9,7 @@
     "esModuleInterop": true,
     "allowSyntheticDefaultImports": true,
     "forceConsistentCasingInFileNames": true,
-    "lib": ["es6"],
+    "lib": ["es6", "dom"],
```

Comment:
Is this needed? if not, perhaps we can remove it.

Reply:
In track1, we don't want to change the generated codes manally, except you have changed it in codegen side. So if it doesn't have any influence, please keep it.